c++/src/kj/filesystem-disk-unix.c++

```diff
@@ -1705,12 +1705,15 @@ private:
       }
     }
 
+    // TODO(someday): Use MAX_PATH from <limits.h>?
+    // TODO(someday): Use std::filesystem::current_path() when compiling with
```

Comment:
KJ generally prefers to avoid the C++ standard library and go direct to OS APIs, so I'd remove this comment.